CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ include(ROCMInstallSymlinks)
 include(CheckCXXCompilerFlag)
 
 rocm_check_target_ids(DEFAULT_AMDGPU_TARGETS
-  TARGETS "gfx908:xnack-;gfx90a:xnack-;gfx90a:xnack+" )
+  TARGETS "gfx908:xnack-;gfx90a:xnack-;gfx90a:xnack+;gfx940:xnack-;gfx940:xnack+" )
```

Comment:
@mkarunan for the targets, let's just set gfx940 without xnack. It will default to xnack:any which is sufficient for now